ex2_hal/iris/hardware_interface/include/iris.h

```diff
@@ -41,7 +42,8 @@
 #define IRIS_POWER_CYCLE_DELAY vTaskDelay(pdMS_TO_TICKS(1))
 #define IRIS_INIT_DELAY vTaskDelay(pdMS_TO_TICKS(4000))
 #define IRIS_WAIT_FOR_STATE_TRANSITION vTaskDelay(pdMS_TO_TICKS(100))
-#define IRIS_IMAGE_DATA_BLOCK_TRANSFER_DELAY vTaskDelay(pdMS_TO_TICKS(10)) // Depends on optimization level
+#define IRIS_IMAGE_DATA_BLOCK_TRANSFER_DELAY vTaskDelay(pdMS_TO_TICKS(20)) // Depends on optimization level
```

Comment:
Depends on who's optimization level

Reply:
The optimization level of the OBC's compiler, i have removed the comment and will test out the data transfer at o3 to see if any modifications is necessary. Basically whatever delay that is chosen will work with the o3 level